### PR TITLE
Update python dependencies

### PR DESCRIPTION
**Pull request by cblecker**

### CODE REVIEW

**Comment by cblecker** [comment]
@stevekuznetsov @Katharine Oh, I saw that you both took care of this, this morning. Do we want to bump further, or just leave it where you have?

File: gubernator/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Jinja2==2.10.1
```
Comment (fejta) [inline gubernator/requirements.txt]: Why does this differ from hack? Are you sure we need a specific version here?

**Comment by cblecker** [comment]
@fejta I.. couldn't find anything referencing or using `hack/test_requirements.txt`. gubernator has it's own, different test_requirements, and nothing in hack touches jinja.

**Comment by Katharine** [comment]
I am somewhat disinclined to do this now, primarily because I spent my entire morning getting gubernator to behave again after bumping the one dependency to a fixed version and have no particular desire to do it again today.

To my knowledge, we have no pressing reason to do this.

**Comment by cblecker** [comment]
@Katharine I'm okay with it.. the security issue is resolved by what's already in the repo. WDYT about just dropping the `hack/test_requirements.txt` file? Do you know of any use for it?

**Comment by fejta** [comment]
> Do you know of any use for it?

Yes we need it to pass unit tests

**Comment by cblecker** [comment]
works for me. I'll just close this :)

**Pull request closed without merging**